openbas-api/src/test/java/io/openbas/rest/exercise/ExerciseApiStatusTest.java

```diff
@@ -176,28 +184,49 @@ void manualStartExerciseTest() throws Exception {
     // -- PREPARE--
     ExerciseUpdateStatusInput input = new ExerciseUpdateStatusInput();
     input.setStatus(ExerciseStatus.RUNNING);
-
-    // -- EXECUTE --
-    String response =
-        mvc.perform(
-                put(EXERCISE_URI + "/" + SCHEDULED_EXERCISE.getId() + "/status")
-                    .content(asJsonString(input))
-                    .contentType(MediaType.APPLICATION_JSON)
-                    .accept(MediaType.APPLICATION_JSON))
-            .andExpect(status().is2xxSuccessful())
-            .andReturn()
-            .getResponse()
-            .getContentAsString();
-
-    // -- ASSERT --
-    Thread.sleep(2000);
-    List<ExecutableInject> injects = injectHelper.getInjectsToRun();
-    Instant nextMinute = now().truncatedTo(MINUTES).plus(1, MINUTES);
-    assertEquals(nextMinute.toString(), JsonPath.read(response, "$.exercise_start_date"));
-    assertEquals(
-        Arrays.asList(ExerciseStatus.CANCELED.name(), ExerciseStatus.PAUSED.name()),
-        JsonPath.read(response, "$.exercise_next_possible_status"));
-    assertEquals(1, injects.size());
+    Instant instantBeforeSetRun = REFERENCE_TIME;
+    Instant instantAfterSetRun = REFERENCE_TIME.plus(1, MINUTES);
+    Clock clock = Clock.fixed(REFERENCE_TIME, ZoneId.of("UTC"));
+
+    try (MockedStatic<Instant> mockedInstant = mockStatic(Instant.class, CALLS_REAL_METHODS)) {
+      try (MockedStatic<Clock> mockedClock = mockStatic(Clock.class, CALLS_REAL_METHODS)) {
+
+        mockedInstant.when(Instant::now).thenReturn(instantBeforeSetRun);
+        // we need this other mock because the production code
+        // inconsistently calls Instant.now() and LocalDateTime.now()
+        mockedClock.when(Clock::systemDefaultZone).thenReturn(clock);
```

Comment:
question: isn't there a risk of timezone mismatch if we mock a systemDefaultZone that isn't UTC by the UTC clock defined l189?

Reply:
All mocked times are specifically UTC